README.md

```diff
@@ -9,31 +9,28 @@ Learn more about this project at - http://sap.github.io/fundamental/
 v1.2.0
 ````
 
-# The Project Structure
-
-* dist - distribution folder that contains combined and individual CSS files for all component
-* docs - Static Documentation site for the library
-* icons - Custom icon set that is used in some components
-* ops - various configurations and gulp tasks for processing scss, building icons, generating documentation site
-* scss - SCSS source files for all the components
-* test - Testing framework for the components
-
-
 # Project Requirements
-
-This project packages the library for `npm` and generates the documentation as a static website.
-
-We rely on several core technologies.
+This project has the following prerequisits and works best with the noted versions.
 
 * Git (for downloading this repo)
-* [Node](https://nodejs.org/) (minimum v6.4)
-* [Gulp](https://gulpjs.com/) (for various ops tasks)
+* [Node LTS](https://nodejs.org/) 8.12.0
```

Comment:
@saadmhybris ~ It might be nice to specifically call out the "Latest" version troubles that occur with Node somewhere. Maybe a "common bugs" section of our readme eventually.